src/ServiceManagement/ServiceBus/ServiceBus.Tests/ClientTest.cs

```diff
@@ -73,6 +70,49 @@ public void CanCreateNamespace()
             }
         }
 
+
+        [Fact]
+        public void CanCreateEventHubNamespace()
+        {
+            using (UndoContext context = UndoContext.Current)
+            {
+                context.Start();
+                var sbClient = TestBase.GetServiceClient<ServiceBusManagementClient>(new RDFETestEnvironmentFactory());
+
+                string namespaceName = TestUtilities.GenerateName("testeventHubNS");
+                string location = sbClient.GetServiceBusRegionsAsync(CancellationToken.None).Result.First().FullName;
+                var response = sbClient.Namespaces.CreateNamespaceAsync(
+                    namespaceName,
+                    new ServiceBusNamespaceCreateParameters
+                    {
+                        Region = location,
+                        CreateACSNamespace = false,
+                        NamespaceType = NamespaceType.EventHub
+                    },
+                    new CancellationToken()).Result;
+
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                Assert.Equal(namespaceName, response.Namespace.Name);
+                Assert.Equal(location, response.Namespace.Region);
+                Assert.Equal(false, response.Namespace.CreateACSNamespace);
+                Assert.Equal(NamespaceType.EventHub, response.Namespace.NamespaceType);
+                int retries = 0;
+                while (true)
+                {
+                    Thread.Sleep(TimeSpan.FromSeconds(4));
```

Comment:
Please wrap this with the if statement where you check that the current mode is not a PlayBack. just add
if (HttpMockServer.Mode != HttpRecorderMode.Playback)
{